src/options/quantifiers_options.toml

```diff
@@ -1000,6 +1000,21 @@ header = "options/quantifiers_options.h"
   default    = "true"
   help       = "use approach to repair constants in sygus candidate solutions"
 
+[[option]]
+  name       = "sygusRepairConstTimeout"
+  category   = "regular"
+  long       = "sygus-repair-const-timeout=N"
+  type       = "unsigned long"
+  help       = "timeout (in milliseconds) for the satisfiability check to repair constants in sygus candidate solutions"
+
+[[option]]
```

Comment:
I wouldn't add this option (otherwise we could want to duplicate every option for the repair-const case). Instead, you can use --ext-rew-prep as a command line parameter, which should be passed automatically to the subcall.